views/partials/screen.handlebars

```diff
@@ -1,4 +1,4 @@
-	<tr class='screen' id='screen-{{{id}}}' data-id='{{id}}' data-filter='{{toLower id}} {{toLower name}}{{#tags}} {{toLower .}}{{/tags}}'>
+	<tr class='screen' id='screen-{{{id}}}' data-id='{{id}}' {{#if type}}data-viewer-type="{{type}}"{{/if}} data-filter='{{toLower id}} {{#if type}}type:{{type}}{{/if}} {{toLower name}}{{#tags}} {{toLower .}}{{/tags}}'>
```

Comment:
I don't think the filter command uses fuzzy searching, meaning that I would have to write the type of screen out first and then the name in order to locate a specific screen.